### PR TITLE
Adds Xeno Corporate Voidsuit Sprites & Modkits

### PR DESCRIPTION
**Pull request by RustingWithYou**
Adds sprites for an Unathi version of the Hephaestus Caiman voidsuit and a Skrell version of the Zeng-Hu Dragon voidsuit helmet, to reflect the close ties that these species have with their respective corporations. They are shown here with the human versions for comparison
image
image

### CODE REVIEW

**Comment by RustingWithYou** [comment]
!review

File: code/game/objects/items/devices/modkit.dm [hunk 1]
```diff
@@ -314,6 +314,34 @@
 		/obj/item/clothing/suit/space/void/unathi_pirate/captain = /obj/item/clothing/suit/space/void/unathi_pirate/hiskyn/captain
 	)
 
+/obj/item/voidsuit_modkit/heph_unathi
```
Comment (Generalcamo) [inline code/game/objects/items/devices/modkit.dm:317]: We have a refitting system. Is this needed?

File: code/game/objects/items/devices/modkit.dm [hunk 2]
```diff
@@ -314,6 +314,34 @@
...
+	suit_options = list(
+		/obj/item/clothing/head/helmet/space/void/hephaestus = /obj/item/clothing/head/helmet/space/void/hephaestus/unathi,
+		/obj/item/clothing/suit/space/void/hephaestus = /obj/item/clothing/suit/space/void/hephaestus/unathi,
+		/obj/item/clothing/head/helmet/space/void/hephaestus/unathi = /obj/item/clothing/head/helmet/space/void/hephaestus,
+		/obj/item/clothing/suit/space/void/hephaestus/unathi = /obj/item/clothing/suit/space/void/hephaestus
+	)
+
+/obj/item/voidsuit_modkit/zeng_skrell
+	name = "zeng-hu pharmaceuticals voidsuit kit"
+	desc = "A highly complicated device that allows you to convert a Zeng-Hu Dragon suit into its Skrell-fitted counterpart and vice versa. Practical!"
```
Comment (Generalcamo) [inline code/game/objects/items/devices/modkit.dm:333]: Ditto

File: code/game/objects/items/devices/modkit.dm (hunk 1 above)
Comment (RustingWithYou) [reply code/game/objects/items/devices/modkit.dm:317]: There is not a cycler for these voidsuits and their status as uplink items means that I want refitting them to be accessible to antags.

File: code/game/objects/items/devices/modkit.dm (hunk 2 above)
Comment (RustingWithYou) [reply code/game/objects/items/devices/modkit.dm:333]: Ditto

File: code/game/objects/items/devices/modkit.dm [hunk 3]
```diff
@@ -314,6 +314,34 @@
 		/obj/item/clothing/suit/space/void/unathi_pirate/captain = /obj/item/clothing/suit/space/void/unathi_pirate/hiskyn/captain
 	)
 
+/obj/item/voidsuit_modkit/heph_unathi
+	name = "hephaestus industries voidsuit kit"
+	desc = "A highly complicated device that allows you to convert a Hephaestus Caiman suit into its Unathi-fitted counterpart and vice versa. Practical!"
+	desc_info = "This is an OOC item, don't let anyone see it! In order to convert a voidsuit simply click on voidsuit or helmet with this item \
```
Comment (SleepyGemmy) [inline code/game/objects/items/devices/modkit.dm:320]:
```suggestion
	desc_info = "This is an OOC item, don't let anyone see it! In order to convert a voidsuit simply click on voidsuit or helmet with this item. \
```

File: code/game/objects/items/devices/modkit.dm [hunk 4]
```diff
@@ -314,6 +314,34 @@
 		/obj/item/clothing/suit/space/void/unathi_pirate/captain = /obj/item/clothing/suit/space/void/unathi_pirate/hiskyn/captain
 	)
 
+/obj/item/voidsuit_modkit/heph_unathi
+	name = "hephaestus industries voidsuit kit"
```
Comment (SleepyGemmy) [inline code/game/objects/items/devices/modkit.dm:318]:
```suggestion
	name = "\improper Hephaestus Industries voidsuit kit"
```

File: code/game/objects/items/devices/modkit.dm [hunk 5]
```diff
@@ -314,6 +314,34 @@
...
+	w_class = ITEMSIZE_SMALL
+	suit_options = list(
+		/obj/item/clothing/head/helmet/space/void/hephaestus = /obj/item/clothing/head/helmet/space/void/hephaestus/unathi,
+		/obj/item/clothing/suit/space/void/hephaestus = /obj/item/clothing/suit/space/void/hephaestus/unathi,
+		/obj/item/clothing/head/helmet/space/void/hephaestus/unathi = /obj/item/clothing/head/helmet/space/void/hephaestus,
+		/obj/item/clothing/suit/space/void/hephaestus/unathi = /obj/item/clothing/suit/space/void/hephaestus
+	)
+
+/obj/item/voidsuit_modkit/zeng_skrell
+	name = "zeng-hu pharmaceuticals voidsuit kit"
```
Comment (SleepyGemmy) [inline code/game/objects/items/devices/modkit.dm:332]:
```suggestion
	name = "\improper Zeng-Hu Pharmaceuticals voidsuit kit"
```

File: code/game/objects/items/devices/modkit.dm [hunk 6]
```diff
@@ -314,6 +314,34 @@
...
+	desc = "A highly complicated device that allows you to convert a Zeng-Hu Dragon suit into its Skrell-fitted counterpart and vice versa. Practical!"
+	desc_info = "This is an OOC item, don't let anyone see it! In order to convert a voidsuit simply click on voidsuit or helmet with this item \
```
Comment (SleepyGemmy) [inline code/game/objects/items/devices/modkit.dm:334]:
```suggestion
	desc_info = "This is an OOC item, don't let anyone see it! In order to convert a voidsuit simply click on voidsuit or helmet with this item. \
```

File: code/modules/clothing/spacesuits/void/misc.dm [hunk 7]
```diff
@@ -359,6 +371,20 @@
...
+/obj/item/clothing/head/helmet/space/void/hephaestus/unathi
+	name = "unathi caiman drop suit helmet"
+	desc = "A massively heavy helmet, part of a larger terraforming suit assembly. This one looks to have been refitted for an Unathi wearer."
+	species_restricted = list(BODYTYPE_UNATHI)
+	icon_state = "caimanhelm_unathi"
+	item_state = "caimanhelm_unathi"
+
+/obj/item/clothing/suit/space/void/hephaestus/unathi
+	name = "caiman drop suit"
+	desc = "A superheavy Hephaestus designed terraforming suit, iconic for its usage in orbital drops onto hostile jungle worlds. This one looks to have been refitted for an Unathi wearer."
```
Comment (SleepyGemmy) [inline code/modules/clothing/spacesuits/void/misc.dm:383]:
```suggestion
	desc = "A superheavy Hephaestus-designed terraforming suit, iconic for its usage in orbital drops onto hostile jungle worlds. This one looks to have been refitted for an Unathi wearer."
```

**Review by SleepyGemmy — APPROVED** [review]

**Review by SleepyGemmy — APPROVED** [review]

**Review by NonQueueingMatt — APPROVED** [review]

**Pull request merged**